package.json

```diff
@@ -32,6 +32,9 @@
     "lodash.findindex": "^4.6.0",
     "lodash.isempty": "^4.4.0",
     "masonry-layout": "^4.2.2",
+    "overlayscrollbars": "^1.12.0",
```

Comment:
Please remove these npm packages. This should be fixed with CSS only

Reply:
Done @brenoferreira

Reply:
@brenoferreira it's working fine with just adding two CSS properties height: 100vh and `overflow-y: scroll` on the top of div wrapper
but it's not that robust as `npm` package!

Reply:
We don't need custom scrollbars nor anything else that justifies loading hundreds of KBs in extra NPM packages. That has a high cost for users loading the page

Reply:
Ohkk Noted!